.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
What's this doing?

Reply:
Those were files added by intellij that I removed

Reply:
Can we remove this `.gitignore` then? It's not needed for the project, right?

Reply:
Yes, it's not needed, you can remove it

Reply:
Can you remove the file from your machine, commit and push again? The commit will get appended this PR itself and the file will not be a part of this PR anymore 🙂

Reply:
I've done that
Do I need a fresh PR?

Reply:
Nope. You can see the changes has been reflected in this PR 🙂